lib/movie.dart

```diff
@@ -0,0 +1,15 @@
+class Movie {
+
+  final int id;
+  final String title;
+  final String originalTitle;
+  final List<String> genres;
+  final String releaseDate;
+  final String voteAverage;
+  final String overview;
+  final String backdrop;
+  final String poster;
+
+  Movie(this.id, this.title, this.originalTitle, this.genres, this.poster, this.backdrop,
+      this.releaseDate, this.voteAverage, this.overview);
+}
```

Comment:
missing end of file extra line, this is configurable from file > settings >editor > general > ensure end of file line on saved (or something similar)